src/main/webapp/js/map.js

```diff
@@ -20,15 +20,13 @@ function createMap(){
 }
 
 function createUfoSightingsMap(){
-    fetch('/ufo-data').then(function(response) {
-      return response.json();
-    }).then((ufoSightings) => {
-
+  fetch('/ufo-data').then(function(response) {
+    return response.json();
+  }).then((ufoSightings) => {
     const map = new google.maps.Map(document.getElementById('map'), {
```

Comment:
LGTM, one question though.  Since this map doesn't rely on the ufoSightings variable would it be better to just declare it independent of the fetch?  It might change behavior for if the fetch call fails, to generate an empty map or no map at all (I'm actually not 100% sure myself since I don't work in JS very often).

Reply:
The ufoSightings variable is the output from ufoDataSevlet, which contains all the information of the ufo sites. So the map actually relies on the ufoSightings variable.